### ISSUE TITLE
Add DateTime Picker for date fields

### ISSUE THREAD

**Issue by digitalmediumltd** [seem-less/gather-town-bulk-changes#7]
Is there a way you could add a date-time picker for fields that require date and times such as synchronized start time for video objects or the display start and end for objects in general? Ideally when saving the form, the dates and times would then converted to Unix timestamps.

**Comment by seem-less** [comment]
yeah there seems to be a way in the JSON Forms API (https://jsonforms.io/examples/control). I don't have much spare time to spend on this project but it should be a relatively easy fix.
